.github/workflows/e2e-timings.yml

```
@@ -0,0 +1,53 @@
+name: E2E - Generate Timings
+
+on:
+  # run this workflow every night at 3am
+  schedule:
+    - cron: '0 3 * * *'
```

@github doesn't recommend starting cron schedules on the hour.

Instead, we should pick an offset after the hour starts, so there's a lower risk of the job getting dropped/delayed: https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#schedule

```suggestion
    - cron: '28 3 * * *'
```